src/test/java/org/xmlcml/cml/element/main/CMLAtomTypeListTest.java

```diff
@@ -0,0 +1,42 @@
+package org.xmlcml.cml.element.main;
```

Comment:
Please add a copyright/license statement. See for example the `CMLArrayListTest.java` file in this repository.